packages/toolpad-app/src/server/auth.ts

```diff
@@ -13,6 +13,8 @@ import { asyncHandler } from '../utils/express';
 import { adaptRequestFromExpressToFetch } from './httpApiAdapters';
 import type { ToolpadProject } from './localMode';
 
+globalThis.crypto ??= (await import('node:crypto')) as Crypto;
```

Comment:
Can this go in https://github.com/mui/mui-toolpad/blob/master/packages/toolpad-app/src/server/index.ts

Reply:
Ok, I've added a comment too explaining why we have the polyfill https://github.com/mui/mui-toolpad/pull/3174/commits/62ac5f3f559b9b5c27667a5e41e2dc276f8ff950